generatebundlefile/main.go

```diff
@@ -142,6 +151,74 @@ func cmdPromote(opts *Options) error {
 	return nil
 }
 
+func cmdRegion(opts *Options) error {
+	BundleLog.Info("Starting Region Check Process")
+	if opts.bundleFile == "" {
+		BundleLog.Info("Please use the --bundle flag when running region check")
+		os.Exit(1)
+	}
+	Bundle, err := ValidateBundle(opts.bundleFile)
+	if err != nil {
+		BundleLog.Error(err, "Unable to validate input file")
+		os.Exit(1)
+	}
+
+	BundleLog.Info("Getting list of images to Region Check")
+	imagesToCheck := make(map[string]string)
+	for _, packages := range Bundle.Spec.Packages {
+		for _, versions := range packages.Source.Versions {
+			for _, images := range versions.Images {
+				imagesToCheck[images.Repository] = images.Digest
+			}
+		}
+	}
+	printMap(imagesToCheck)
+
+	//STS and ECR Client to get Account Number
+	BundleLog.Info("Creating SDK connections to Region Check")
+	clients := &SDKClients{}
+	clients, err = clients.GetProfileSDKConnection("ecr", "default", ecrRegion)
+	if err != nil {
+		BundleLog.Error(err, "getting SDK connection")
+		os.Exit(1)
+	}
+	clients, err = clients.GetProfileSDKConnection("sts", "default", ecrRegion)
+	if err != nil {
+		BundleLog.Error(err, "getting profile SDK connection")
+		os.Exit(1)
+	}
+	var imagesNotFound []string
+	for _, region := range RegionList {
+		BundleLog.Info("Starting Check for", "Region", region)
+		clients, err = clients.GetProfileSDKConnection("ecr", "default", region)
+		if err != nil {
+			BundleLog.Error(err, "getting ECR SDK connection")
+			os.Exit(1)
+		}
+		if err != nil {
+			BundleLog.Error(err, "error creating ECR client")
+			os.Exit(1)
+		}
```

Comment:
second block is dead code

Reply:
good catch thanks

Reply:
fixed in [4a4cbb8](https://github.com/aws/eks-anywhere-packages/pull/668/commits/4a4cbb808e64bfb0970cd9753e31b606e66a714e)